doc/integrator/install_application.rst

```diff
@@ -3,21 +3,17 @@
 Install an existing application
 ===============================
 
-On this page we explain all the procedures to build an application from
+On this page we explain the procedure to build an application from
 only the code.
 
-For example If you want to use an existing database you should ignore
+If you want to use an existing database, you should ignore
 all the commands concerning the database.
 
-This guide considers that:
- - We use a server manages by Camptocamp, meaning:
-    - all dependencies described in the
-      :ref:`requirements <integrator_requirements>` are installed,
-    - Postgres has a GIS template ``template_postgis`` and a user ``www-data``,
```

Comment:
Why removing this line?

Reply:
Because later in the instructions, it says what to do if you don't have template_postgis

Reply:
And it also says what to do if do not want to use "www-data" as user